src/wp-admin/options-writing.php

```diff
@@ -168,11 +170,21 @@
 <th scope="row"><label for="mailserver_login"><?php _e( 'Login Name' ); ?></label></th>
 <td><input name="mailserver_login" type="text" id="mailserver_login" value="<?php form_option( 'mailserver_login' ); ?>" class="regular-text ltr" /></td>
 </tr>
-<tr>
-<th scope="row"><label for="mailserver_pass"><?php _e( 'Password' ); ?></label></th>
-<td>
-<input name="mailserver_pass" type="text" id="mailserver_pass" value="<?php form_option( 'mailserver_pass' ); ?>" class="regular-text ltr" />
-</td>
+<tr class="mailserver-pass-wrap">
+	<th scope="row">
+		<label for="mailserver_pass">
+			<?php _e( 'Password' ); ?>
+		</label>
+	</th>
+	<td>
+		<input class="hidden" value=" " /><!-- #24364 workaround -->
```

Comment:
These were changed to `type="hidden"` in [changeset 55892](https://core.trac.wordpress.org/changeset/55892)